packages/runtime/package.json

```diff
@@ -58,6 +58,11 @@
       "import": "./dist/esm/next/index.js",
       "types": "./dist/types/next/index.d.ts"
     },
+    "./next/middleware": {
+      "require": "./dist/cjs/next/middleware/index.js",
```

Comment:
I wonder if we should drop support for CJS and only support ESM for middleware 🤔